test/e2e/extensions/ms-ceintl.vscode-language-pack-es-1.70.0/package.json

```diff
@@ -28,5 +28,11 @@
         ]
       }
     ]
+  },
+  "__metadata": {
+    "id": "47e020a1-33db-4cc0-a1b4-42f97781749a",
+    "publisherDisplayName": "MS-CEINTL",
+    "publisherId": "0b0882c3-aee3-4d7c-b5f9-872f9be0a115",
+    "isPreReleaseVersion": false
```

Comment:
Was this necessary to fix something?  Might be sketchy to include stuff that comes from the closed-source marketplace.

Reply:
Oh somehow missed your comment where you said VS Code added it.  In that case seems fine to me!